boa/contracts/vyper/vyper_contract.py

```diff
@@ -117,13 +117,13 @@ def stomp(self, address: Any, data_section=None) -> "VyperContract":
         return ret
 
     # TODO: allow `env=` kwargs and so on
-    def at(self, address: Any) -> "VyperContract":
+    def at(self, address: Any, nowarn=False) -> "VyperContract":
```

Comment:
IMO more specific is better
```suggestion
    def at(self, address: Any, check_bytecode=True) -> "VyperContract":
```